src/proxy/main.c

```diff
@@ -0,0 +1,118 @@
+#include <errno.h>
+#include <getopt.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "libhirte/bus/bus.h"
+#include "libhirte/common/common.h"
+#include "libhirte/common/opt.h"
+#include "libhirte/log/log.h"
+
+const struct option options[] = { { ARG_HELP, no_argument, 0, ARG_HELP_SHORT }, { NULL, 0, 0, '\0' } };
+
+#define OPTIONS_STR ARG_HELP_SHORT_S
+
+static const char *opt_node_unit = NULL;
+
+static void usage(char *argv[]) {
+        printf("Usage: %s \n"
+               "\t[-h help]\t\t Print this help message.\n"
+               "\t<node>_<service>\t Input the name of the node the required service is supposed to run on.\n",
```

Comment:
I guess this should have two underscores between node and service.

Reply:
Or alternatively we could use just one. It seems like systemd normaly uses dashes to separate words, like "abrt-journal-core.service", and a single dash won't conflict with that.

Reply:
Only exceptions on my machine are:

- proc-sys-fs-binfmt_misc.automount
- proc-sys-fs-binfmt_misc.mount,
- var-lib-nfs-rpc_pipefs.mount
- wpa_supplicant.service
- rpc_pipefs.target

So, probably fine.

Reply:
I changed this to use a single dash